### PR TITLE
RLP-97162: Added trusted Login ip List Functionality

### PR DESCRIPTION
**Pull request by ndhemiwal**
## Types of changes

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes if appropriate.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/data-sources/trusted_login_ip.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+page_title: "Prisma Cloud: prismacloud_trusted_login_ips"
```
Comment (nikitapanw) [inline docs/data-sources/trusted_login_ip.md:2]: should be prismacloud_trusted_login_ip

File: docs/data-sources/trusted_login_ips.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Example Usage
+
+```hcl
+data "prismacloud_trusted_login_ips" "example" {}
+```
+
+## Attribute Reference
+
+* `total` - (int) Total number of account groups.
```
Comment (nikitapanw) [inline docs/data-sources/trusted_login_ips.md]: Make it Total number of trusted login ips

File: docs/resources/trusted_login_ip.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+Manage a Login IP Allow List.
+
+## Example Usage
+
+```hcl
+resource "prismacloud_trusted_login_ip" "example" {
+  name = "My new List"
+  description = "Made by Terraform"
+  cidr = [
+    "199.167.52.0/24"
```
Comment (nikitapanw) [inline docs/resources/trusted_login_ip.md]: Give ip address as 1.1.1.1 don't put 199.167.52.0/24

File: docs/data-sources/trusted_login_ip.md (hunk 1 above)
Comment (ndhemiwal) [reply docs/data-sources/trusted_login_ip.md:2]: done

**Pull request merged**